cli_test.go

```diff
@@ -38,6 +39,7 @@ func TestCLILoad(t *testing.T) {
 		actualSettings, err := cliProvider.Load()
 		if err != nil {
 			t.Error(err)
+			return err
```

Comment:
I think we should just switch the `t.Error` calls to `t.Fatal`

Reply:
Do I need to remove `return err` if changes to `t.Fatal`

Reply:
Probably for the best - it becomes unreachable after the change anyway.

Reply:
Hi, I did the changes as your suggestion. Please check.